src/components/create-board-modal/create-board-modal.module.css

```
@@ -22,7 +30,10 @@
   box-shadow: rgba(0, 0, 0, 0.1) 2px 2px 0px 1px;
   font-size: var(--body-font-size);
   line-height: 16.003px;
-  z-index: 1000;
+  z-index: 1;
+  color: inherit;
+  margin: 0;
```

Stylelint (17.12.0)

[error] Overridden property "margin-left" by shorthand "margin" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)